doc/concepts/morphology.rst

```diff
@@ -504,9 +504,10 @@ interpret SWC files similarly to how the NEURON simulator would, and how the All
 
 Despite the differences between the interpretations, there is a common set of checks that are always performed
 to validate an SWC file:
-   * Check that there are no duplicate ids.
-   * Check that the parent id of a sample is less than the id of the sample.
-   * Check that the parent id of a sample refers to an existing sample.
+
```

Comment:
Those exact changes are in the Decor [PR](https://github.com/arbor-sim/arbor/pull/1235/files). I think they can be removed to avoid the rebase issues.

Reply:
I'll remove it.